fixtures/nextest-tests/proc-macro-test/src/lib.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this an extra newline? Could be removed if not necessary.

Reply:
This file contains only one newline character.

While I am not sure if `rustc` requires the source file to always end with new line separator(C/C++ somewhat requires this), I personally think it's good to follow that convention. For more rationale see https://unix.stackexchange.com/a/18789. But I don't feel very strong of it. I am Ok with leaving an actual empty file. WDYT?

Reply:
Ah. So I love trailing newlines, and I definitely agree that non-empty files should have them. But logically, an empty file has no lines, so there's no line to have a trailing newline appended to.

In other words, a file can be treated as a list of lines:

```
foo\n
bar\n
baz\n
```

and trailing newlines make sense because it's effectively part of the last line. But an empty file has no lines, so a trailing newline doesn't logically fit in.

Reply:
Done.